desktop/mac/release-notes/index.md

```diff
@@ -33,6 +33,46 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### New
+
+- Released Docker Desktop for Linux.
```

Comment:
Could we have a link to the Linux docs?

Reply:
Sure, is this link ok? https://docs.docker.com/desktop/linux/

Reply:
Yes, thanks @lorenrh